docs/docs/download.md

```diff
@@ -20,18 +20,33 @@ Because the app is not signed, you will need to right-click, or hold the <kbd>co
 
 ## Download for Linux
 
-### Ubuntu 18.04 and newer
+### Flatpak
+
+1. [Set up Flatpak](https://www.flatpak.org/setup/)
+
+2. Install xemu from [Flathub](https://flathub.org/apps/details/org.xemu.xemu)
+
+`flatpak install -y org.xemu.xemu`
+
+3. Run xemu
+
+`flatpak run org.xemu.xemu`
+
+Note: Only the `~/.var/app/app.xemu.xemu/data/xemu/xemu` directory can be written by xemu.
+The Hard Disk image has to be placed there, for example at `~/.var/app/app.xemu.xemu/data/xemu/xemu/xbox_hdd.qcow2`.
+
+### Ubuntu PPA
```

Comment:
Please revert ubuntu section header change. I still recommend Ubuntu users just use the ppa

Reply:
How is it related? I've changed from `Ubuntu 18.04 and newer` to `Ubuntu PPA`.
It can be confusing to keep the old title because it assumes Flatpak is not compatible with Ubuntu 18.04 or newer, which is false.
I'll precise the version in parenthesis.